runtime/jcl/common/com_ibm_oti_vm_VM.cpp

```diff
@@ -21,15 +21,17 @@
  *******************************************************************************/
 
 #include <string.h>
-#include "jni.h"
 #include "jcl.h"
 #include "jclglob.h"
 #include "jclprots.h"
-#include "jcl_internal.h"
+#include "jni.h"
+#include "omrlinkedlist.h"
 #include "util_api.h"
+#include "VMHelpers.hpp"
 
 /* private static native byte[][] getVMArgsImpl(); */
 
+extern "C" {
```

Comment:
Please add a blank blank line after line 34.

Reply:
Also, the comment on line 32 should not be separated from the function it relates to.